examples/symlink-write.js

```diff
@@ -1,25 +1,28 @@
 var fstream = require('../fstream.js')
 var notOpen = false
 
-fstream
-  .Writer({
-    path: 'path/to/symlink',
-    linkpath: './file',
-    isSymbolicLink: true,
-    mode: '0755' // octal strings supported
-  })
-  .on('close', function () {
-    notOpen = true
-    var fs = require('fs')
-    var s = fs.lstatSync('path/to/symlink')
-    var isSym = s.isSymbolicLink()
-    console.log((isSym ? '' : 'not ') + 'ok 1 should be symlink')
-    var t = fs.readlinkSync('path/to/symlink')
-    var isTarget = t === './file'
-    console.log((isTarget ? '' : 'not ') + 'ok 2 should link to ./file')
-  })
-  .end()
+// No symlinks on Windows
```

Comment:
I partially agree - unix-like symlinks that have been available since Vista probably aren't worth the effort, since they require special permissions and/or elevation. Additionally windows requires you to specify the type of the target (file of directory) when the symlink is created, which is weird.

Junctions are more limited, but they're universally available and don't require special permissions. For most intents and purposes (e.g. NPM) they're good to use, and I would advise to do so. There are some caveats though:
  * Junctions can only point at directories.
  * Junctions always have to contain absolute paths.
  * Node internally resolves the target path when a junction is created with a relative path, however in most node versions (up to v0.10, and even in v0.12 I believe) there's a bug in it. See https://github.com/nodejs/node-v0.x-archive/pull/8813
  * Very old & unsupported versions of windows (XP prior to SP1 I believe) had some serious issues where a junction would be traversed when recursively deleting a directory. But nobody uses that any more, not even in China.
  * Node treats junctions like symlinks for most intents and purposes; `stat`, `lstat`, `readlink`, `realpath` etc. all deal with them properly.